libs/application/ui-fields/src/lib/DividerFormField.tsx

```diff
@@ -1,15 +1,23 @@
 import React, { FC } from 'react'
```

Comment:
This file is the main concern for you @arnifreyrs

Reply:
I like this